packages/cozy-doctypes/src/Document.js

```diff
@@ -75,7 +81,29 @@ class Document {
     }
   }
 
+  static unregisterClient() {
+    this.cozyClient = null
+  }
```

Comment:
Why is there an unregister ?

Since the client is registered on all the class, it can be dangerous since if you launch parallel queries on different cozies (like ACH batch), you can think that it is safe, to register a client multiple times, when it is not.

Reply:
yeah, it's for the tests (because doing Document.`registerClient(null)` was not working (anymore?) Will change it to `dangerousUnregisterClient` and add a comment

Reply:
I am not fond of having a such method, even because of the tests. AFAIR, Jest has ways to reset modules that maintain a state, so you would be able to start over without this method

Reply:
(Speaking about that, I see more and more statefull objects and modules, like this class with everything static but still an internal state. This is generally an anti-pattern)

Reply:
Sorry @edas I saw your comment after the merge. I did a PR to remove the method: https://github.com/cozy/cozy-libs/pull/459